perfmetrics/scripts/job_files/seq_rand_read_write_presubmit.fio

```diff
@@ -0,0 +1,119 @@
+[global]
```

Comment:
Can we add comment here (or document somewhere) how much time roughly it takes for the performance test on one branch and the total time it takes for complete this presubmit test once ?

Reply:
added in build.sh script.